action/protocol/account/util/util.go

```diff
@@ -54,7 +54,21 @@ func LoadOrCreateAccount(sm protocol.StateManager, encodedAddr string) (*state.A
 }
 
 // LoadAccount loads an account state
-func LoadAccount(sr protocol.StateReader, addrHash hash.Hash160) (*state.Account, error) {
+func LoadAccount(sr protocol.StateReader, addr address.Address) (*state.Account, error) {
+	var account state.Account
+	addrHash := hash.BytesToHash160(addr.Bytes())
```

Comment:
`return LoadAccountByHash160(addrHash)`

Reply:
Ahh..  Done

Reply:
@dustinxie Review comments incorporated

Reply:
thx, would you like to pick up https://github.com/iotexproject/iotex-core/issues/2775 as well?